_posts/2020-09-17-strategy-change.md

```diff
@@ -0,0 +1,9 @@
+---
+title: 17 September 2020 - Improved communication strategy
+categories: [news, release]
+tweet: We are changing gears on twitter! Instead of tweeting on every commit, we'll send less frequent but more interesting tweets: important releases, new features and other contributions! Make sure to check https://github.com/fieldtrip, and http://www.fieldtriptoolbox.org/discussion_list.
+---
+
+### 17 September, 2020
+
+We are changing our [communication](/development/guideline/communication) strategy, which is an integral part of our work! We will continue to use this website for news items and updates on releases, but also use twitter more systematically. Instead of sending a tweet upon every commit (which made sense in the pre-GitHub days but not any more), we will start sending less frequent but more interesting tweets with important releases, announcements of new features, documentation updates, training events, and other interesting information! So to follow the important news, please keep checking here or follow us on [twitter](https://twitter.com/fieldtriptoolbx).
```

Comment:
In addition to 'systematically', I'd say that twitter is going to be used more to-the-point

Reply:
resolved in https://github.com/fieldtrip/website/pull/288/commits/8deb759fea53e521835c3552d96f00bc79b92407